h/models/group.py

```diff
@@ -34,6 +34,18 @@ class WriteableBy(enum.Enum):
     members = "members"
 
 
+class GroupMembership(Base):
+    __tablename__ = "user_group"
+
+    __table_args__ = (sa.UniqueConstraint("user_id", "group_id"),)
+
+    id = sa.Column("id", sa.Integer, autoincrement=True, primary_key=True)
+    user_id = sa.Column("user_id", sa.Integer, sa.ForeignKey("user.id"), nullable=False)
+    group_id = sa.Column(
+        "group_id", sa.Integer, sa.ForeignKey("group.id"), nullable=False
+    )
```

Comment:
This should be [imported into `models/__init__.py`](https://github.com/hypothesis/h/blob/3b76115fcf7d09d9ac1849237575757dc46ef0dc/h/models/__init__.py#L20-L38) if you want to import it. `from h.models import GroupMembership` rather than `from h.models.group import GroupMembership`. It's more convenient and also means that the sub-organization of the `h/models/` dir isn't part of its public interface

Reply:
Righto, I'll add it there.

Reply:
Dun